File: Search Insert Position/Search_Insert_Position.py
class Solution:
    def searchInsert(self, nums: List[int], target: int) -> int:
        for i in range(0, len(nums),1):
            if (target != nums[i]):
                if nums[i] > target:
                    return i
            else:
                return i
        return len(nums)
            

File: Search Insert Position/Search_Insert_Position.java
class Solution {
    public int searchInsert(int[] nums, int target) {
        for (int i = 0; i < nums.length ; i++) {
            if (nums[i] != target) {
                if (nums[i] > target) {
                    return i;
                }
            }else return i;
        }
        return nums.length;
    }
}